addons/account/models/res_bank.py

```diff
@@ -38,7 +38,14 @@ def build_qr_code_vals(self, amount, free_communication, structured_communicatio
                 error_message = self._check_for_qr_code_errors(candidate_method, amount, currency, debtor_partner, free_communication, structured_communication)
 
                 if not error_message:
-                    return candidate_method, amount, currency, debtor_partner, free_communication, structured_communication
+                    return {
+                        'qr_method': candidate_method,
+                        'amount': amount,
+                        'currency': currency,
+                        'debtor_partner': debtor_partner,
+                        'free_communication': free_communication,
+                        'structured_communication': structured_communication,
+                    }
```

Comment:
docstring of this method is still the old from build_qr_code_url => can you change that as well?

Reply:
and why was it not introduced as a private method?

Reply:
> docstring of this method is still the old from build_qr_code_url => can you change that as well?

Done :+1: 

> and why was it not introduced as a private method?

My bad, I changed it :+1:

Reply:
on a side note: we shouldn't change the API on stable version (so changing a method name shouldn't be done), but I suppose we can consider that OK since the method was added few days ago only and it was broken since then :shrug:

Reply:
I had the same thought :slightly_smiling_face: